test/core/transport/binder/end2end/fuzzers/BUILD

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//bazel:grpc_build_system.bzl", "grpc_package")
+
+grpc_package(
+    name = "test/core/transport/binder/end2end/fuzzers",
+    features = [
+        "layering_check",
+    ],
+)
+
+licenses(["notice"])
+
+load("//test/core/util:grpc_fuzzer.bzl", "grpc_fuzzer")
```

Comment:
This load need to be moved to the top of the file

Reply:
Other than this, copybara import looks ok

Reply:
> This load need to be moved to the top of the file

Done.

> Other than this, copybara import looks ok

Nice, thanks!